Sources/GoogleMapsUtils/Clustering/Algorithms/GridBasedCluster/GMUGridBasedClusterAlgorithm1.swift

```diff
@@ -58,19 +58,22 @@ final class GMUGridBasedClusterAlgorithm1: GMUClusterAlgorithm1 {
         var clusters: [Int : GMUCluster1] = [:]
 
         // Divide the whole map into a numCells x numCells grid and assign items to them.
-        let numCells: Double = Double(ceil(256 * pow(2, zoom) / Float(gmuGridCellSizePoints)))
+        let numCells: Int = Int(ceil(256 * pow(2, zoom) / gmuGridCellSizePoints))
 
         for item in clusterItems {
             let point: GMSMapPoint = GMSProject(item.position)
             /// point.x is in [-1, 1] range
-            let col: Double = numCells * (1.0 + point.x) / 2
+            let col: Int = Int(Double(numCells) * (1.0 + point.x) / 2.0)
             /// point.y is in [-1, 1] range
-            let row: Double = numCells * (1.0 + point.y) / 2
-            let index: Int = Int(numCells * row + col)
+            let row: Int = Int(Double(numCells) * (1.0 + point.y) / 2.0)
```

Comment:
[self-review]: `row`, `col` must be `Int`, hence updated the type from `Double`.